frontend/src/config/mozambique/prism.json

```diff
@@ -24,7 +24,8 @@
   },
   "defaultDisplayBoundaries": [
     "admin_boundaries",
-    "admin1_boundaries"
+    "admin1_boundaries",
```

Comment:
Small suggestion to invert admin1 and admin2 here to have admin boundaries in a logical order

Reply:
done, though we should also remove the requirement that the default be called admin_boundaries. This would be clearer if we could name it admin3_boundaries.

Reply:
this was captured a long time ago here https://github.com/WFP-VAM/prism-app/issues/360